nex/matchmake-extension/update_notification_data.go

```diff
@@ -1,32 +1,46 @@
 package nex_matchmake_extension
 
 import (
-	"fmt"
+	"log"
 
 	nex "github.com/PretendoNetwork/nex-go"
 	"github.com/PretendoNetwork/wiiu-chat-secure/database"
 	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
+	"github.com/PretendoNetwork/wiiu-chat-secure/grpc"
 	nex_notifications "github.com/PretendoNetwork/wiiu-chat-secure/nex/notifications"
 
 	matchmake_extension "github.com/PretendoNetwork/nex-protocols-go/matchmake-extension"
+	"github.com/PretendoNetwork/nex-protocols-go/notifications"
 )
 
 func UpdateNotificationData(err error, client *nex.Client, callID uint32, uiType uint32, uiParam1 uint32, uiParam2 uint32, strParam string) {
-	// TODO: Implement this
-	fmt.Printf("uiType: %d, uiParam1: %d, uiParam2: %d, strParam: %s\r\n", uiType, uiParam1, uiParam2, strParam)
+	log.Printf("uiType: %d, uiParam1: %d, uiParam2: %d, strParam: %s\r\n", uiType, uiParam1, uiParam2, strParam)
+	recipientClient := globals.NEXServer.FindClientFromPID(uiParam2)
+
+	if uiType == 101 {

```

Comment:
Type 101 is [defined in `nex-protocols-go`](https://github.com/PretendoNetwork/nex-protocols-go/blob/master/notifications/notification_categories.go#L25), please use the constant